server/common/annotations/hosted_tiledb.py

```diff
@@ -93,20 +107,20 @@ def convert_to_pandas_df(self, tileDBArray):
         return new_df
 
     def write_labels(self, df, data_adaptor):
-
         user_id = current_app.auth.get_user_id()
+        user_name = current_app.auth.get_user_name().replace(' ', '')
```

Comment:
Also why remove spaces?  I'm not sure if there is a current need, but that would make it harder to map from uri to user name.

Reply:
It looks like adding spaces to file names in s3 causes some downloading issues/isnt recommended
https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html

I can replace it with an underscore if that would be clearer

Reply:
discussed over slack